### PR TITLE
docs: add lerna to showcase

### PR DESCRIPTION
**Pull request by bcabanes**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).

## Motivation

It adds the Lerna website (https://lerna.js.org) to the showcase section of Docusaurus' website.

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1142,6 +1142,15 @@ const Users: User[] = [
     source: 'https://github.com/leon-ai/docs.getleon.ai',
     tags: ['opensource', 'versioning', 'i18n'],
   },
+  {
+    title: 'Lerna',
+    description:
+      'A fast modern build system for managing and publishing multiple JavaScript/TypeScript packages from the same repository.',
+    preview: require('./showcase/lerna.png'),
+    website: 'https://lerna.js.org',
+    source: 'https://github.com/lerna/lerna',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1151]: Is the _website_ open source? I can't find it anywhere.

If not, remember to [remove the "Edit this page" links](https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs#editUrl) at the bottom of the pages as they currently point to our repo.

Comment (slorber) [reply website/src/data/users.tsx:1151]: couldn't either

btw your editUrl is wrong ;)

Comment (bcabanes) [reply website/src/data/users.tsx:1151]: Thank you for the reviews! 
The edit URL is updated!

We are in the process of migrating the website to Lerna's monorepo. Here is the PR https://github.com/lerna/lerna/pull/3187, it should be done soon.

Comment (Josh-Cena) [reply website/src/data/users.tsx:1151]: Great—let's wait for that PR to be merged first then. No hurry.

Comment (bcabanes) [reply website/src/data/users.tsx:1151]: The PR has been merged 👍

**Review by Josh-Cena — APPROVED** [review]
Thanks!

Great to see Lerna using Docusaurus—and Docusaurus using Lerna

**Comment by Josh-Cena** [comment]
Ah, wait—when you force-push you forgot to pull my changes. Now I have to do it again.

**Pull request merged**